ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -42,22 +42,31 @@ import { AddressError } from '../../models/addressError';
 import { AddressVerificationStatus } from '../../models/addressVerificationStatus';
 import { AddressInputComponent } from './addressInput.component';
 import { MailAddressBlock } from '../../models/activity/MailAddressBlock';
-import { generateTaggedAddress, isStreetRequiredError } from './addressUtils';
+import { generateTaggedAddress } from './addressUtils';
 import { SubmitAnnouncementService } from '../../services/submitAnnouncement.service';
 import { AddressVerificationWarnings } from '../../models/addressVerificationWarnings';
 import { NGXTranslateService } from '../../services/internationalization/ngxTranslate.service';
 import { LoggingService } from '../../services/logging.service';
 import { ConfigurationService } from '../../services/configuration.service';
 
+interface FormError {
+    message: string;
+    isEasyPostError: boolean;
+}
+
+interface FieldError extends AddressError {
+    isEasyPostError: boolean;
+}
```

Comment:
extend all types of errors with `isEasyPostError` flag in order to divide errors and calculate `isEasyPostInvalid` value

Reply:
Suggestion: create IsEasyPostError interface and  have form error and FieldError extend from it so you can use them uniformly, e.g.:
```typescript
interface IsEasyPostError {
    isEasyPostError: boolean;
}

interface FormError extends IsEasyPostError {
    message: string;
}

export interface FieldError extends AddressError, IsEasyPostError {}

Reply:
good idea, done!